frontend/components/campaigns/dashboard/CouponHelpDialog.tsx

```diff
@@ -0,0 +1,57 @@
+import { Stack, Typography } from '@mui/material';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { CampaignPublicData } from '../../../types/campaigns';
+import Button from '../../generic/Button';
+
+interface Props {
+  open: boolean;
+  campaign: CampaignPublicData;
+  setIsCouponHelpOpen: (isOpen: boolean) => void;
+}
+
+const CouponHelpDialog = ({ open, campaign, setIsCouponHelpOpen }: Props) => {
+  const numTotalCoupons = campaign.promisedAmount / campaign.couponDenomination;
+
+  return (
+    <Dialog open={open} onClose={() => setIsCouponHelpOpen(false)}>
+      <DialogTitle>
+        <Typography variant="h1">More about coupon distribution</Typography>
+      </DialogTitle>
+
+      <DialogContent>
```

Comment:
(The explanation doesn't make sense because it starts by explaining the sponsorship process and not answering what coupons are. Also saying the coupons are split into $10 each while saying 50 coupons in a separate line is disjointed and makes people do the mental math again.)

<img width="639" alt="Screenshot 2022-11-03 at 12 22 40 PM" src="https://user-images.githubusercontent.com/71375383/199647181-b5ce1595-9cf8-4d5b-ae05-61264487912c.png">

**Suggested change**

Coupons are generated from a sponsor's donation.

In this campaign, Alice Tan has sponsored $500. GIVING COUPONS INLINE LOGO split the amount into 50 coupons at $10 each. The coupons were distributed to the public. Coupon recipients can choose which charity to donate the $10 to and they can add a personal contribution if they wish.